workers_launch_template.tf

```diff
@@ -351,6 +351,43 @@ resource "aws_launch_template" "workers_launch_template" {
     }
   }
 
+  dynamic "block_device_mappings" {
+    for_each = lookup(var.worker_groups_launch_template[count.index], "additional_volumes", lookup(local.workers_group_defaults, "additional_volumes", []))
+    content {
+      device_name = block_device_mappings.value.block_device_name
+
+      ebs {
```

Comment:
Does it make sense to rename `local.additional_volumes` into `local. additional_ebs_volumes` ? It sounds that we are only dealing with EBS and not ephemeral storage.

@jaimehrubiks @js-timbirkett @max-rocket-internet any thoughts ?

Reply:
I don't really have an opinion in this matter, both alternatives look good for me.

Reply:
accepted as makes sense